### PR TITLE
Add new options and SIGINT handling

### PR DESCRIPTION
**Pull request by johvet**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -53,30 +53,46 @@ module.exports = function(options, cb) {
...
     var stream = combiner(logs, formatter);
     stream.logStream = logs;
 
+    process.on('SIGINT', () => {
+      console.log(`Received SIGINT. Asking ECS to stop task: ${taskId}`);
+
+      let params = {
+        clusterArn: options.clusterArn,
+        taskId: taskId,
+        reason: 'User cancelled tasked'
```
Comment (ziggythehamster) [inline index.js]: I think this should be `User requested interrupt`

File: lib/taskrunner.js [hunk 2]
```diff
@@ -26,6 +26,25 @@ module.exports = {
...
+
+    if (options.env !== undefined) {
+      params.overrides.containerOverrides[0].environment = options.env;
+    }
+
     ecs.runTask(params, cb);
+  },
+
+  stop: function(options, cb) {
+    let ecs = new AWS.ECS();
```
Comment (ziggythehamster) [inline lib/taskrunner.js]: This could be a const

File: lib/taskrunner.js [hunk 3]
```diff
@@ -26,6 +26,25 @@ module.exports = {
...
+    let params = {
```
Comment (ziggythehamster) [inline lib/taskrunner.js]: This could be a const

File: index.js [hunk 4]
```diff
@@ -53,30 +53,46 @@ module.exports = function(options, cb) {
...
     });
-    
+
     var stream = combiner(logs, formatter);
     stream.logStream = logs;
 
+    process.on('SIGINT', () => {
+      console.log(`Received SIGINT. Asking ECS to stop task: ${taskId}`);
+
+      let params = {
```
Comment (ziggythehamster) [inline index.js]: This could be a const

File: index.js [hunk 5]
```diff
@@ -27,7 +27,7 @@ module.exports = function(options, cb) {
 
   async.waterfall([
     function(next) {
-      var ecs = new AWS.ECS();
+      let ecs = new AWS.ECS();
```
Comment (ziggythehamster) [inline index.js]: This could be a const.

**Review by ziggythehamster — CHANGES_REQUESTED** [review]

**Review by jcaffey — APPROVED** [review]

**Review by ziggythehamster — APPROVED** [review]

**Pull request merged**